subsys/nrf_security/src/drivers/cracen/cracenpsa/src/key_management.c

```diff
@@ -1232,7 +1232,7 @@ psa_status_t cracen_get_builtin_key(psa_drv_slot_number_t slot_number,
 	switch (slot_number) {
 	case CRACEN_IDENTITY_KEY_SLOT_NUMBER:
 		psa_set_key_lifetime(attributes, PSA_KEY_LIFETIME_FROM_PERSISTENCE_AND_LOCATION(
-							 PSA_KEY_PERSISTENCE_READ_ONLY,
```

Comment:
If this is intended to flag that a key can be imported but not erase then it should not be used for the IKG keys since they cannot be imported, right?

Reply:
I fail to follow. I've changed this to match this with KMU. If you follow the text of the define it states

```
/*
 * Defines a persistence state where the key can't be erased
 * In this state the key will only be erased if ERASEALL is available and run
 */
 #define CRACEN_KEY_PERSISTENCE_READ_ONLY 0x03
```

Reply:
Sure but this type can be imported for KMU but it cannot get imported for IKG keys right? So this doesn't strictly have a rule on if they key could be imported or not. The PSA_KEY_PERSISTENCE_READ_ONLY states that the keys cannot be imported, right?

Reply:
Even though we support generating the boot seed (which the IKG is is based on) on the device, there is technically nothing preventing you from generating that seed off-chip and importing it. In this case `psa_import_key` would naturally be extended to support this unless we create a fully proprietary API only for this, similar to any other key stored in KMU.